packages/kit/src/exports/index.js

```diff
@@ -8,6 +8,10 @@ import { HttpError, Redirect, ValidationError } from '../runtime/control.js';
  * @param {any} message
  */
 export function error(status, message) {
+	if ((!__SVELTEKIT_BROWSER__ || __SVELTEKIT_DEV__) && (status < 400 || status > 599)) {
```

Comment:
I assume you did this to safe some bytes in the client bundle - should we do the same as part of this PR for `redirect`?

Reply:
good catch, done